etl/steps/data/garden/worldbank_wdi/2023-05-29/wdi.meta.yml

```diff
@@ -1985,33 +1984,28 @@ tables:
           title_public: Share of the population using the Internet
           # title_variant: WDI
           title_variant: International Telecommunication Union
-          attribution_short: the World Bank
           # attribution: # defaults to origin[*].producer so does not have to be set here
           topic_tags:
-          - Technological Progress
+          - Technological Change
           - Internet
-          - Loneliness and Social Connections
+          - Loneliness & Social Connections
           faqs: []
           grapher_config:
-            slug: share-of-individuals-using-the-internet
```

Comment:
@danyx23 is `slug` and `columnSlug` useless? I ask because `slug` is in JSON schema as a valid field `Slug of the chart on Our World in Data`.

Reply:
@Marigold yeah it is and it is potentially misleading. The grapher db in the charts table is the authoritative source on grapher slugs and using slugs here is at best harmless and at worst might lead to weird bugs